whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -193,7 +196,9 @@ public RdapObject mapError(final int errorCode, final String errorTitle, final L
     }
 
     public RdapObject mapHelp(final String requestUrl) {
-        return mapCommons(new RdapObject(), requestUrl);
+        RdapObject rdapObject =  mapCommons(new RdapObject(), requestUrl);
+        rdapObject.setRdapConformance(COMMON_RDAP_CONFORMANCE);
```

Comment:
In /help I am adding FLAT_MODEL as well. Maybe is better to take all the values that we have in RdapConformance, because in /help we will always show the supported profiles that we have, so if in the future we need to add another one we just need to add it in RdapCOnformance and it will automatically add the value in /help

Reply:
OK but use the addAll not setter as it will replace the previously added RdapConformance.